### PR TITLE
Add support for Django 2.2

### LINKED ISSUE #48: Add support for Django 2.2
Lets add support for the new LTS version of Django, 2.2.

### PR DESCRIPTION
**Pull request by bengerman13**
Remove support for django 1.x, 2.0

Fixes #48 
Maybe obsoletes #40

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -98,7 +98,7 @@ def run(self):
       include_package_data=True,
       packages=find_packages(),
       install_requires=[
-          'django>=1.8,<2.1',
+          'django>=2.1,<3.0',
```
Comment (tbaxter-18f) [inline setup.py]: How do we feel about just pinning to 2.2.x, since it's the LTS version?

Comment (bengerman13) [reply setup.py]: seems reasonable to me

**Comment by tbaxter-18f** [comment]
We need to update the docs and uaa_client VERSION to note this is a 2.0 release, breaking change, and if you're on django 1.x you need to pin to the older 1.3 release, but the code part of it looks good. Thanks!

**Comment by bengerman13** [comment]
> We need to update the docs and uaa_client VERSION to note this is a 2.0 release, breaking change
The version stuff should happen on a separate commit, according to the [release docs](https://github.com/18F/cg-django-uaa/blob/master/RELEASE.md)

> if you're on django 1.x you need to pin to the older 1.3 release

I'm a little worried that suggesting this will imply that we're maintaining a 1.x cycle for django 1.x. I'll throw something in with a warning tone to it :)

**Comment by tbaxter-18f** [comment]
Once we figure out the failing tests, LGTM!

**Pull request merged**